sdk/tools/azure-openrewrite/pom.xml

```diff
@@ -11,8 +11,48 @@
   <name>Microsoft Azure OpenRewrite Plugin library for Java</name>
   <description>This module contains OpenRewrite recipe for migrating to next generation Microsoft Azure client libraries.</description>
 
+  <distributionManagement>
+    <site>
+      <id>azure-openrewrite</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <url>https://github.com/azure/azure-sdk-for-java</url>
+  <organization>
+    <name>Microsoft Corporation</name>
+    <url>http://microsoft.com</url>
+  </organization>
+  <licenses>
+    <license>
+      <name>The MIT License (MIT)</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>microsoft</id>
+      <name>Microsoft Corporation</name>
+    </developer>
+  </developers>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/Azure/azure-sdk-for-java/issues</url>
+  </issueManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:https://github.com/Azure/azure-sdk-for-java.git</connection>
+    <developerConnection/>
+    <tag>HEAD</tag>
+  </scm>
+
   <properties>
     <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
+    <site.url>https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java</site.url>
```

Comment:
Does this link still work?

Reply:
I'm not sure 😄 

I'm just copying prior art throughout the repository. We should investigate this as a follow-up as its larger than this PR.